src/EventListener/LoadDataContainerListener.php

```diff
@@ -12,9 +12,6 @@
 
 class LoadDataContainerListener
 {
-    /**
-     * @Hook("loadDataContainer")
```

Comment:
was not used because it's registered in `config.php` and annotations are not supported with contao 4.4